supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -12,7 +12,7 @@
 [discrete]
 [[samba]]
 ==== image:images/yes.png[yes] Samba (noun)
-*Description*: "Samba" is a freeware program that allows end users to access and use files, printers, and other commonly shared resources on a company's intranet or on the internet. Samba can be installed on a variety of operating system platforms, including Linux, most common Unix platforms, OpenVMS, and OS/2.
+*Description*: "Samba" is a freeware program that allows end users to access and use files, printers, and other commonly shared resources on a company's intranet or on the internet. Samba can be installed on a variety of operating system platforms, including Linux, most common UNIX platforms, OpenVMS, and OS/2.
```

Comment:
Can we re-phrase to implement active tense? For example: 
_You can install Samba on a variety of opearting syetms......_

Reply:
Maybe this could be done in another PR if necessary - to keep this one clean to just the UNIX updates

Reply:
Agreed!